### DISCUSSION TITLE
Allow other screen orientations in godot editor for Android

### DISCUSSION THREAD

**Discussion by dariocohen** [godotengine/godot-proposals#9144] [Platforms]
Hi, I'm currently extensively using the Godot editor on my Android phone to develop a classic shmup type game. Having a full time job and doing everyday adult-life stuff leaves me with little time to sit at a desk and work for my hobby projects. I have found out that implementing workflows that utilizes the more than sufficient capabilities of the device that I'm always carrying around has been essential in keeping things flowing.

One thing that I always appreciate is being able to use apps in portrait mode. Since it is the most used orientation of most apps, I find it frustrating to be forced to physically rotate my phone multiple times while cycling through apps with different forced orientation modes. Sadly, this is the case with the Godot Editor in android as it is forced in landscape mode.

![WhatsApp Image 2024-02-21 at 11 17 02](https://github.com/godotengine/godot-proposals/assets/22178851/3dccaba2-2ca8-46c1-a267-c1a772928dce)
I must say that I tried to accommodate myself in working with the editor in this mode but it's really not a pleasant experience to do any kind of text editing in a touch keyboard that wide.

Tinkering around I've found that splitting the screen (a feature first introduced in android 7.0) allows to bypass the forced orientation and this enables some really cool perks that considerably improved my workflow and the editor usability in general on the phone

1. The bottom window prevents the touch keyboard to be shown on top of the Godot editor. This is essential while editing gdscript code and any type of text editing
2. I can run my game along side the editor and even step through breakpoints while debugging
3. I can look up documentation, YouTube videos,  or Google stuff up while working on my game. I usually have termux running below for moving files around and commiting code to git.

https://github.com/godotengine/godot-proposals/assets/22178851/c844039f-dfa7-4dd2-bf0a-2af8d3ed2585

All these things are very cool but there's still a thing that irks me. Whenever I open up the godot engine the screen forces itself to the landscape orientation that I have no use for. I consider that it would be better and more natural to have the app follow the orientation of the device or to have an editor option to set up the orientation of the app to accommodate for more types of workflows.

https://github.com/godotengine/godot-proposals/assets/22178851/26b39010-c164-445d-91e0-234fffc21397

**Comment by theludovyc** [comment]
Hello 👋

Maybe you can be interested by this issue https://github.com/godotengine/godot-proposals/issues/10140 😊

**Comment by Sammyueru** [comment]
bump

**Reply by Calinou** [reply]
Please don't bump issues without contributing significant new information. Use the :+1: reaction button on the first post instead.
